pkg/agent/openflow/pipeline.go

```diff
@@ -679,6 +681,7 @@ func (f *featurePodConnectivity) conntrackFlows() []binding.Flow {
 				MatchProtocol(ipProtocol).
 				MatchCTStateNew(true).
 				MatchCTStateTrk(true).
+				MatchCTStateSNAT(false).
```

Comment:
why this is required?

Reply:
yes, it's for skipping another DNAT commit.